R/calculate.scaling.R

```diff
@@ -1,21 +1,21 @@
 calculate.scaling <- function(data.matrices) {
 	# if there is only one data type to scale
-	if (class(data.matrices)[1] == 'matrix') {
+	if (class(data.matrices)[1] == 'matrix' || class(data.matrices)[1] == 'data.frame') {
 		# return the mean and sd of each row
 		return(list(
 			center = apply(data.matrices,1,mean),
 			scale = apply(data.matrices,1,sd)
 			));
+		} else if (class(data.matrices) == 'list') {
```

Comment:
I'm not sure why `else if` is used here instead of the original `if`. Can you elaborate on it?

Reply:
And while you're at it, put the constant on the left side of comparison operator too please :)

Reply:
@maotian06 I still see non-yoda conditions in other R scripts. Can you find and update them throughout for consistency as Paul suggested?

Reply:
I see!!! I changed it back! @tyamaguchi-ucla